src/plugins/elasticsearch/lib/check_es_version.js

```diff
@@ -13,8 +12,8 @@ module.exports = function (server) {
   .then(function (info) {
     var badNodes = _.filter(info.nodes, function (node) {
       // remove client nodes (Logstash)
-      var isClient = _.get(node, 'attributes.client');
-      if (isClient != null && esBool(isClient) === true) {
+      var attrs = _.get(node, 'attributes');
+      if (attrs && attrs.master === 'false' && attrs.data === 'false') {
```

Comment:
does it make sense to use `esBool(attrs.master)`, `esBool(attrs.data)` here?  I'm not sure if it's ever anything other than 'false' but under the assumption the method was made for a reason